tests/Operation/WatchFunctionalTest.php

```diff
@@ -322,20 +379,67 @@ public function testNoChangeAfterResumeBeforeInsert()
         $this->assertMatchesDocument($expectedResult, $changeStream->current());
     }
 
-    public function testResumeTokenIsUpdatedAfterResuming()
+    public function testResumeMultipleTimesInSuccession()
```

Comment:
This method was introduced in #621 for [PHPLIB-442](https://jira.mongodb.org/browse/PHPLIB-442) earlier this week, so I didn't feel too bad about renaming it. Refactoring it to _also_ check for key increments (in addition to the existing result assertions) seemed preferable to duplicating the entire test to only check key increments.

I also added some additional resume cases, which weren't in the original test.